resources/developer-guides/daemon-rpc.md

```diff
@@ -842,35 +983,7 @@ $ curl http://127.0.0.1:18081/json_rpc -d '{"jsonrpc":"2.0","id":"0","method":"g
       "seconds": 28
     }],
     "status": "OK"
-  }
-}
-```
-
-
-### **flush_txpool**
-
-Flush tx ids from transaction pool
```

Comment:
Why is this one removed ?

Reply:
It gave me an error when trying to call it, but i guess that could be intended. Reverting